application/src/main/java/org/opentripplanner/updater/siri/updater/lite/SiriETLiteHttpTripUpdateSource.java

```diff
@@ -0,0 +1,93 @@
+package org.opentripplanner.updater.siri.updater.lite;
+
+import static org.opentripplanner.updater.trip.UpdateIncrementality.FULL_DATASET;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.Optional;
+import org.opentripplanner.framework.io.OtpHttpClientException;
+import org.opentripplanner.updater.siri.updater.EstimatedTimetableSource;
+import org.opentripplanner.updater.siri.updater.SiriFileLoader;
+import org.opentripplanner.updater.siri.updater.SiriLoader;
+import org.opentripplanner.updater.spi.HttpHeaders;
+import org.opentripplanner.updater.trip.UpdateIncrementality;
+import org.opentripplanner.utils.tostring.ToStringBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.org.siri.siri20.Siri;
+
+/**
+ * SIRI Light downloads periodically all messages as a single GET request.
+ */
+public class SiriETLiteHttpTripUpdateSource implements EstimatedTimetableSource {
+
+  private static final Logger LOG = LoggerFactory.getLogger(SiriETLiteHttpTripUpdateSource.class);
+  /**
+   * The framework code requires a requestor ref but in SIRI Light this is not used.
+   */
+  private static final String DUMMY_REQUESTOR_REF = "OpenTripPlanner";
+
+  private final Parameters parameters;
+
+  private final SiriLoader siriLoader;
+
+  public SiriETLiteHttpTripUpdateSource(Parameters parameters) {
+    this.parameters = parameters;
+
+    this.siriLoader = createLoader(parameters);
+  }
+
+  @Override
+  public Optional<Siri> getUpdates() {
+    try {
+      var siri = siriLoader.fetchETFeed(DUMMY_REQUESTOR_REF);
```

Comment:
This dummy parameter is a little bit of a code smell. But it seems to be the simplest solution.

Reply:
An alternative would be to make the parameter nullable.

Reply:
The issue seems to be that it's required for one implementation and not required for the other implementations. So making it nullable isn't 100% accurate either. IMO sending a dummy value is a good enough solution.